humidity/pxt.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "jacdac-humidity",
+    "version": "0.0.0",
+    "description": "A sensor measuring humidity of outside environment.",
+    "files": [
+        "constants.ts",
+        "client.g.ts"
+    ],
+    "testFiles": [
+        "test.ts"
+    ],
+    "supportedTargets": [
```

Comment:
maybe we skip supportedTargets unless necessary? (ie the client requires some libraries that are say missing in microbit)